src/services/fetchers/ByBitFetcher.ts

```diff
@@ -0,0 +1,103 @@
+import axios from 'axios';
+import {inject, injectable} from 'inversify';
+import {Logger} from 'winston';
+
+import Settings from '../../types/Settings.js';
+
+type FeedValue = {fsym: string; tsym: string};
+
+export type InputParams = Map<string, FeedValue>;
+
+export interface OutputValue {
+  fsym: string;
+  tsym: string;
+  value: number;
+}
+
+@injectable()
+class ByBitFetcher {
```

Comment:
let's add to `getContainer()` as singelton, unfortunately we often forget about it

Reply:
Done